learnrl/callbacks/tensorflow.py

```diff
@@ -0,0 +1,52 @@
+import importlib
+
+tensorflow_spec = importlib.util.find_spec('tensorflow')
+
+if tensorflow_spec is not None:
```

Comment:
Il faudrait faire un test qui test ce qui se passe lorsque tensorflow_spec is None mais que l'on appelle TensorboardCallback ! (Que le message d'erreur dise bien qu'il faut installer tensorflow>=version)

Reply:
Je vais regarder comment effectuer les tests

Il faut quelle version en particulier pour Tensorflow ? Parce que pour l'instant, j'ai juste mis "Tensorflow should be installed"

Reply:
Je dirais >= 2.0.0 mais en vrai ça va surtout dépendre du TensorflowEstimator / KerasEstimator